frontend/src/services/storage/storage.service.ts

```diff
@@ -0,0 +1,29 @@
+type Constructor = {
+  storage: globalThis.Storage;
+};
+
+class Storage {
+  #storage: globalThis.Storage;
+
+  constructor({ storage }: Constructor) {
+    this.#storage = storage;
+  }
+
+  public getItem(key: string): string | null {
+    return this.#storage.getItem(key);
+  }
+
+  public setItem(key: string, value: string): void {
+    return this.#storage.setItem(key, value);
+  }
+
+  public removeItem(key: string): void {
```

Comment:
Let's use `StorageKey` enum as a type for key arg

Reply:
Good point, let's use it for all Storage methods